libfreerdp/core/gateway/tsg.c

```diff
@@ -670,14 +676,7 @@ static BOOL TsProxyCreateTunnelReadResponse(rdpTsg* tsg, RPC_PDU* pdu,
 
 			case TSG_ASYNC_MESSAGE_REAUTH:
 				{
-					size_t loffset = Stream_GetPosition(pdu->s);
-					UINT32 offset = (UINT32)loffset;
-
-					if (loffset > UINT32_MAX)
-						goto fail;
-
-					if (!Stream_SafeSeek(pdu->s, rpc_offset_align(&offset, 8) ||
-					                     (Stream_GetRemainingLength(pdu->s) < 8)))
```

Comment:
- Don't know the internals but the new tsg_stream_align always uses `4` while we had `8`.
- Not sure if the removed checks were important